src/equations/compressible_navier_stokes_2d.jl

```diff
@@ -123,26 +111,19 @@ struct GradientVariablesEntropy end
 
 # default to primitive gradient variables
 function CompressibleNavierStokesDiffusion2D(equations::CompressibleEulerEquations2D;
-                                             Reynolds, Prandtl, Mach_freestream,
+                                             viscosity, Prandtl,
```

Comment:
I think we should keep some consistency here: the function argument is called `viscosity`, the struct variable is `mu`, and in the elixirs we call it `dynamic_viscosity`. I'd say, we should consistently on either `dynamic_viscosity` (more descriptive) or `mu` (easier to use inside the code).

Reply:
Okay, we can rename it to dynamic_viscosity everywhere. Is it okay if in the `flux` we can still pull the value into a variable like `mu = equations.dynamic_viscosity`? That way the implemented formula matches the paper formula 1-to-1

Reply:
Yes, sounds good :+1:

Reply:
Sure, that works as well. But wouldn't it make sense then to just use `mu` everywhere and save us the hassle of having two different variable names?

Reply:
That would be more consistent with using `gamma` already for Euler, I guess :shrug:

Reply:
Sure that works for me. I can update to use mu everywhere.

Reply:
Done